src/components/Game.js

```diff
@@ -33,13 +34,19 @@ function Game() {
     /* eslint-disable-next-line */
       alert('Game over!'); 
       setIsCounting(false);
+      setRoundOverStage(true);
       setAvailableWords(words);
       setTimeLeft(1 * 60);
     }
   };
 
   const handleStart = () => {
     setIsCounting(true);
+    setCorrectScore(0);
+    nextWord();
```

Comment:
functions should contain a verb in their names because functions are doing something. `nextWord` sounds like a variable that contains data, not a function. a good name would be `setNextWord`, or smth like this